src/app/containers/SheriffProfilePluginLeaves/LeavesFieldTable.tsx

```diff
@@ -110,7 +110,6 @@ export default class LeavesFieldTable extends React.Component<LeavesFieldTablePr
                                     {...p}
                                     nullTimeLabel={nullTimeLabel}
                                     timeIncrement={30}
-                                    style={{ width: 780 }}
```

Comment:
I think there is a similar line that will need to be removed on the Multi Shift Edit Form, which also uses the time picker

Reply:
Looks like there was a default width set (which I removed) and the other one wasn't setting a value for it.

Reply:
👍